moveit_ros/perception/mesh_filter/src/transform_provider.cpp

```diff
@@ -110,17 +110,24 @@ void TransformProvider::run()
   while (!stop_)
```

Comment:
This should probably be:

`while (ros::ok() && !stop_)`

To avoid getting stuck in an infinite loop.

Reply:
Good catch--done